public/app/features/api-keys/ApiKeysForm.tsx

```diff
@@ -99,6 +119,17 @@ export const ApiKeysForm: FC<Props> = ({ show, onClose, onKeyAdded }) => {
                 onChange={onSecondsToLiveChange}
               />
             </div>
+            <div className="gf-form">
+              <label>
+                New Service Account
+                <input
+                  name="createServiceAccount"
+                  type="checkbox"
+                  checked={createServiceAccount}
+                  onChange={onCreateServiceAccountChange}
+                />
```

Comment:
nit: maybe change to more explicit boolean
```suggestion
                <input
                  name="createServiceAccount"
                  type="checkbox"
                  checked={Boolean(createServiceAccount)}
                  onChange={onCreateServiceAccountChange}
                />
```